### ISSUE TITLE
log each error only once and log number of times error happened

### ISSUE THREAD

**Issue by wrashid** [LBNL-UCB-STI/beam#1248]

**Comment by michaz** [comment]
How are we going to achieve this? Remember that loggers are framework infrastructure -- e.g. I want to be able to use ``ActorLogging``, ``LogginFSM`` etc., so forcing code to use a custom logger isn't an option IMO.

**Comment by wrashid** [comment]
yes, nothing would change for how you log, but on logging library level (this feature is not for general beam use, but for a specific application, where we don't want to expose users to all exceptions, but still have a summary)

**Comment by JustinPihony** [comment]
What is the benefit of this? The only way to get this would be to keep track of all messages - or you could cut down with some fuzzy logic. But I assume you still want the logs in near real time?

**Comment by carloscaldas** [comment]
It was created and merged into `competitions-pin`
